enterprise/kernel/src/main/java/org/neo4j/kernel/impl/enterprise/StandardBoltConnectionTracker.java

```diff
@@ -53,7 +53,11 @@ public void onTerminate( ManagedBoltStateMachine machine )
     @Override
     public Set<ManagedBoltStateMachine> getActiveConnections( String owner )
     {
-        return sessions.entrySet().stream().filter( entry -> entry.getValue().equals( owner ) )
+        return sessions
+                .entrySet()
+                .stream()
+                .filter( entry -> entry.getValue()
+                .equals( owner ) )
```

Comment:
This line should probably join it's buddy above, in lack of anything important to say...